js/server/tests/resilience/resilience-synchronous-repl-cluster.js

```diff
@@ -321,6 +322,26 @@ function SynchronousReplicationSuite () {
 ////////////////////////////////////////////////////////////////////////////////
 
     setUp : function () {
+      if (!inquiry_checked) {
+        console.warn("Checking inquiry");
+        var writeResult = ArangoAgency.write([[{"a":1},{"a":{"oldEmpty":true}},"INTEGRATION_TEST_INQUIRY_ERROR_503"]]);
+        console.log("Inquired successfully a matched precondition under 503 response from write");
```

Comment:
`writeResult` and its structure are not tested here. The result structure may be wrong??

Reply:
The result structure is tested in the agency-tests.

Reply:
Why not test here as well? Tests should not rely on other tests IMHO.

Reply:
And the agency tests do not test the interaction that's happening in AgencyComm::sendWithFailover, they directly call the /inquire backend.